.github/workflows/build-test.yml

```
@@ -18,11 +18,19 @@ jobs:
         uses: docker/setup-buildx-action@v2
 
+      - name: Cache Docker layers
+        uses: useblacksmith/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
       - name: Build Docker image with cache
         uses: docker/build-push-action@v4
```

actionlint (1.7.4)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)